adal/versioning/version_tasks.gradle

```diff
@@ -5,7 +5,14 @@ def getVersionFile() {
 def getVersionProps() {
     def versionPropsFile = getVersionFile()
     def versionProps = new Properties()
-    versionProps.load(new FileInputStream(versionPropsFile))
+    def versionPropsStream = new FileInputStream(versionPropsFile)
+
```

Comment:
Maybe something like withStream(versionProsStream, s -> versionProps.load(s) ) ?

https://docs.groovy-lang.org/latest/html/gapi/org/codehaus/groovy/runtime/IOGroovyMethods.html#withStream(U,%20Closure)

Reply:
@AdamBJohnsonx I'll take that tip:
https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/1545